frontend/src/metabase/browse/components/ModelExplanationBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import { Flex, Paper, Icon, Text } from "metabase/ui";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+import { updateSetting } from "metabase/admin/settings/settings";
+import { getSetting } from "metabase/selectors/settings";
+
+import { BannerCloseButton, BannerModelIcon } from "./BrowseModels.styled";
+
+export function ModelExplanationBanner() {
+  const hasDismissedBanner = useSelector(state =>
+    getSetting(state, "dismissed-browse-models-banner"),
+  );
+  const dispatch = useDispatch();
+
+  const [shouldShowBanner, setShouldShowBanner] = useState(!hasDismissedBanner);
+
+  const dismissBanner = () => {
+    setShouldShowBanner(false);
+    dispatch(
+      updateSetting({
+        key: "dismissed-browse-models-banner",
+        value: true,
+      }),
+    );
+  };
+
+  if (!shouldShowBanner) {
+    return null;
+  }
+
+  return (
+    <Paper
+      mt="1rem"
+      mb="-0.5rem"
```

Comment:
@rafpaf I think this gets us the same effect as the `fixPaddingUnderBanner`, but let me know if i'm missing something.